README.md

```diff
@@ -99,16 +99,18 @@ The item in `miq_vm_nics` list of can contain following attributes:
 
 ManageIQ variables:
 
-| Name              | Default value       |  Description                                                               |
-|-------------------|---------------------|----------------------------------------------------------------------------|
-| miq_app_username  | admin               | The username used to login to ManageIQ.                                    |
-| miq_app_password  | smartvm             | The password of user specific in username used to login to ManageIQ.       |
-| miq_username      | admin               | Alias of `miq_app_username` for backward compatibility.                    |
-| miq_password      | smartvm             | Alias of `miq_app_password` for backward compatibility.                    |
-| miq_db_username   | root                | The username to connect to the database.                                   |
-| miq_db_password   | `miq_app_password`  | The password of user specific in username used to connect to the database. |
-| miq_region        | 0                   | The ManageIQ region created in the database.                               |
-| miq_company       | My Company          | The company name of the appliance.                                         |
+| Name               | Default value       |  Description                                                               |
+|--------------------|---------------------|----------------------------------------------------------------------------|
+| miq_app_username   | admin               | The username used to login to ManageIQ.                                    |
+| miq_app_password   | smartvm             | The password of user specific in username used to login to ManageIQ.       |
+| miq_username       | admin               | Alias of `miq_app_username` for backward compatibility.                    |
+| miq_password       | smartvm             | Alias of `miq_app_password` for backward compatibility.                    |
+| miq_db_username    | root                | The username to connect to the database.                                   |
+| miq_db_password    | `miq_app_password`  | The password of user specific in username used to connect to the database. |
+| miq_region         | 0                   | The ManageIQ region created in the database.                               |
+| miq_company        | My Company          | The company name of the appliance.                                         |
+| miq_disabled_roles | []                  | List of ManageIQ roles to disable on the appliance.                        |
```

Comment:
maybe mentioning 'server roles' would be better? or somehow distinguish what roles are those